plugins/destination/s3/client/spec.go

```diff
@@ -40,6 +40,10 @@ func (s *Spec) SetDefaults() {
 		b := true
 		s.TestWrite = &b
 	}
+	if s.EndpointSkipTLSVerify == nil {
+		b := true
+		s.EndpointSkipTLSVerify = &b
```

Comment:
Setting skip to true means it would be skipped by default? Is that what we really want?

Reply:
Definitely not